base/codeintel-db/README.md

```diff
@@ -0,0 +1,8 @@
+## Important Notice
+
+Segment's self-hosted version of Sourcegraph uses an [External RDS Service](https://docs.sourcegraph.com/admin/external_services/postgres#kubernetes) for
+the `codeintel-db`.
+
+You can find the definition for that database in the [segmentio/terracode-tooling](https://github.com/segmentio/terracode-tooling/blob/master/stage/us-west-2/sourcegraph/rds.tf) repository.
```

Comment:
Can you press 'y' on the github page so it points to a sha and not master? That way the link won't break if the file is ever moved or removed?

Reply:
TIL. Thats a super helpful keybind